src/libs/actions/IOU.js

```diff
@@ -534,23 +546,20 @@ function splitBillAndOpenReport(participants, currentUserLogin, amount, comment,
 }
 
 /**
- * Cancels or declines a transaction in iouReport.
- * Declining and cancelling transactions are done via the same Auth command.
- *
  * @param {String} chatReportID
  * @param {String} iouReportID
- * @param {String} type - cancel|decline
- * @param {Object} moneyRequestAction - the create IOU reportAction we are cancelling
+ * @param {Object} moneyRequestAction - the IOU reportAction we are deleting
```

Comment:
Should we drop the IOU terminology here since it can also be expense?
```suggestion
 * @param {Object} moneyRequestAction - the money request reportAction we are deleting
```